calendar_test.go

```diff
@@ -360,3 +361,15 @@ func TestIssue52(t *testing.T) {
 		t.Fatalf("cannot read test directory: %v", err)
 	}
 }
+
+func TestIssue77(t *testing.T) {
+	url := "https://proseconsult.umontpellier.fr/jsp/custom/modules/plannings/direct_cal.jsp?data=58c99062bab31d256bee14356aca3f2423c0f022cb9660eba051b2653be722c4c7f281e4e3ad06b85d3374100ac416a4dc5c094f7d1a811b903031bde802c7f50e0bd1077f9461bed8f9a32b516a3c63525f110c026ed6da86f487dd451ca812c1c60bb40b1502b6511435cf9908feb2166c54e36382c1aa3eb0ff5cb8980cdb,1"
+
+	cal, err := ParseCalendarFromUrl(url)
```

Comment:
This looks like it would be making a real HTTPS request in tests. It should be using a mock http client.